### PR TITLE
Rename/add legacy project information

### PR DESCRIPTION
**Pull request by evelyn9191**
Followed instructions in https://www.python.org/dev/peps/pep-0423/#how-to-rename-a-project

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python3
-from setuptools import setup, find_packages
+from setuptools import setup
 
 setup(
```
Comment (bzamecnik) [inline setup.py:4]: We should probably also bump the version and make a last release.

File: setup.py [hunk 2]
```diff
@@ -26,25 +26,12 @@
...
-        "polling2",
-        "more-itertools",
+        "rossum",
     ],
     python_requires=">=3.6",
-    setup_requires=["pytest-runner"],
-    tests_require=["pytest", "pytest-cov", "requests_mock", "pytest-click"],
     zip_safe=False,
-    entry_points={"console_scripts": ["elisctl = elisctl.main:entry_point"]},
+    entry_points={"console_scripts": ["rossum = rossum.main:entry_point"]},
```
Comment (bzamecnik) [inline setup.py]: The console script is already provided by the rossum package. Probably no need to make it again here.

Also see installer.cfg.

File: setup.py (hunk 1 above)
Comment (evelyn9191) [reply setup.py:4]: That will be done automatically during version bumping (see `Contributing` section in [README](https://github.com/rossumai/rossum))

**Review by bzamecnik — APPROVED** [review]
LGTM

**Pull request merged**